analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -304,19 +304,21 @@ def add_arguments_to_parser(parser):
     analyzer_opts.add_argument('--config',
                                dest='config_file',
                                required=False,
-                               help="Allow the configuration from an explicit "
-                                    "JSON based configuration file. The "
-                                    "value of the 'analyzer' key in the "
+                               help="R|Allow the configuration from an "
```

Comment:
```suggestion
                               help="Allow the configuration from an "
```

Reply:
This is a special syntax. See:
https://github.com/Ericsson/codechecker/blob/dbf5618c00b26f41197d8fa2f1599a3758909924/codechecker_common/arg.py#L23-L33